src/GeoJSONTables.jl

```diff
@@ -101,10 +101,13 @@ function miss(x)
     end
 end
 
+Base.show(io::IO, fc::StructArray) = println(io, "FeatureCollection with $(length(fc)) Features")
```

Comment:
This overwrites the show method that is defined in StructArrays. We shouldn't do that.

Reply:
Oh. I only found one here https://github.com/JuliaArrays/StructArrays.jl/blob/4483b57939f82828be51bc1e87c4a2de5f49ccf5/src/lazy.jl#L19 for LazyRow not StructArray so went ahead with it?

Reply:
AH, found it https://github.com/JuliaArrays/StructArrays.jl/blob/4483b57939f82828be51bc1e87c4a2de5f49ccf5/src/structarray.jl#L223

Reply:
Hmm yeah you maybe right that it is not defined in StructArrays. But still it's not good practice to overload base functions on types you don't own.